src/scene.rs

```diff
@@ -166,6 +188,15 @@ impl<I: ImageSize> Scene<I> {
         total
     }
 
+    /// Get the number of running animations for a sprite. If sprite does not
+    /// exist, returns 0.
```

Comment:
I thought about that but opted against it - zero is "correct" if the sprite doesn't exist, and for the common case I'd prefer not having to unwrap it.